packages/ui/components/AppCard.tsx

```diff
@@ -0,0 +1,33 @@
+import { Protocol, ProtocolBadge } from './ProtocolBadge';
+
+interface AppCardProps {
+  isActive?: boolean;
+  name: string;
+  description: string;
+  protocols: Protocol[]
+  className?: string;
+}
+
+export const AppCard = ({isActive = false, name, description, protocols, className}:AppCardProps) => {
+  return (
+    <>
+      <div
+        className={`bg-gray-800 border-gray-800 rounded-lg w-[523px] border-2 ${className} ${
+          isActive ? ' border-pink-800/30 shadow-service-card' : ''
+        }`}
+      >
+        <div className="flex flex-col gap-2 px-5 py-3">
+          <h3 className="text-base font-medium text-gray-100">{name}</h3>
+          <div className='flex gap-1'>
+            {protocols.map((protocol, index) => (<ProtocolBadge protocol={protocol} key={index} />))}
```

Comment:
We should also show if the app is a client or a server (or both). These icons:

<img width="80" alt="Captura de pantalla 2023-09-06 a las 17 16 23" src="https://github.com/asyncapi/studio/assets/242119/d8c7923c-1151-4366-8eea-aa16e44ec574">

From left to right, these are Server and Client respectively.

So you should probably take two more props in this component: `isClient` and `isServer`. I'd put them in the beginning before the protocols.